lib/workers/repository/process/lookup/__snapshots__/index.spec.ts.snap

```diff
@@ -170,12 +170,10 @@ Object {
 
 exports[`workers/repository/process/lookup/index .lookupUpdates() handles git submodule update 1`] = `
 Object {
-  "currentVersion": undefined,
```

Comment:
Need to check this isn't a mistake

Reply:
It's because you moved the `versioning.valueToVersion` call to inside the `isValid` condition and git submodules don't have a current value.

https://github.com/renovatebot/renovate/blob/9a306048a9f59644974765acf4abf512df5cde2f/lib/workers/repository/process/lookup/index.ts#L57-L57

@JamieMagee Maybe we should pass the tracking branch as current value and add a git versioning, which allows branches / tags as version?

Reply:
So we call `valueToVersion` now only on valid `currentValue`. Maybe we should mark this pr as feature, as it causes a bahavior change here.